lib/markdown-preview-enhanced.coffee

```diff
@@ -249,3 +256,13 @@ module.exports = MarkdownPreviewEnhanced =
     editor = atom.workspace.getActiveTextEditor()
     if editor and editor.buffer
       editor.insertText '<!-- slide -->\n'
+
+  # HOOKS Issue #101
```

Comment:
This 3 code styles are different... 
Modify them in the future as I am too lazy now